### ISSUE TITLE
Implement appropriate editorial changes from WCAG 2.2 update into WCAG2ICT

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#607]
This issue is to track implementation of changes identified in the analysis of #580

### Currently

WCAG 2.2 has made many small editorial changes, which appear neither in WCAG2ICT quotations of WCAG nor in the word substitutions.

I've noticed these differences:

* More links from SCs to glossary terms
* More links from one glossary term to another
* Changes in capitalization, e.g. "Web" to "web"; "Cognitive function test" to "cognitive function test"
* A change in a list structure in SC 2.5.8 Target Size
* Punctuation

### Proposed

* Apply all recent changes from WCAG 2.2 principles, guidelines, SCs, and glossary terms to the WCAG2ICT quotations of these things. (Could we apply these changes in an automated way?)
* Apply similar changes to the word substitution descriptions and to the content after substitution. (Probably requires manual edits, but could we get a detailed list of what changed from the WCAG PRs or some kind of diff?)
* Check WCAG2ICT-only content for similar writing style, e.g. change "Web" to "web" (if any, best effort)

**Comment by ChrisLoiselle** [comment]
@daniel-montalvo we can talk to this issue that @mitchellevan raised either on this upcoming call for this week or next. In general, do you know of any automation that may assist with what he has raised?

**Comment by daniel-montalvo** [comment]
> [@daniel-montalvo](https://github.com/daniel-montalvo) we can talk to this issue that [@mitchellevan](https://github.com/mitchellevan) raised either on this upcoming call for this week or next. In general, do you know of any automation that may assist with what he has raised?

Straightforward solution is for me to run my local script that pulls WCAG content. I will do that and update the current Editors Draft if that's the desire.

We are working on an automated wcag.json file at WAI, I do plan to modify the current WCAG2ICT  build system so that it picks it up automatically, but I cannot garantee when this would be available.

**Comment by daniel-montalvo** [comment]
Hi @mitchellevan , @maryjom @ChrisLoiselle 

I wonder if we do want this process fully automated or we want to have control as to when the updated WCAG content appears in our document. If we fully automate this, chances are that updated WCAG content that has an impact on the existing WCAG2ICT guidance is inserted in the Editor's Draft before the WCAG2ICT Task Force gets a chance to decide whether the guidance needs updating. Just double checking if that's what we want.

**Comment by maryjom** [comment]
@daniel-montalvo IMO, since changes in the WCAG language can cause a mismatch between the quoted content and the WCAG2ICT interpretation content, I think a more manual approach would be better. Otherwise it could be confusing. I already have issue #580 open for us to look at what effects this might have on WCAG2ICT interpretation sections for the SC and definitions where the text has changed. I would prefer to incorporate the change to the quoted content when we've also made sure the interpretation content is also updated. This should all be done with issue 580 work.

**Comment by mitchellevan** [comment]
> [@mitchellevan](https://github.com/mitchellevan) I had an open issue to look at the document for differences that need incorporation (See issue [#580](https://github.com/w3c/wcag2ict/issues/580).) Have you make it through all of the changes in WCAG to determine how the update affects WCAG2ICT language?

I had not noticed #580, that's great. Let's call #580 an analysis task, and #607 an implementation task that depends on #580.

**Comment by maryjom** [comment]
@mitchellevan Works for me!  I'll adjust the issue titles so that's clear.

**Comment by maryjom** [comment]
All of these changes have been merged into the Editor's draft, so closing this issue. 

@mapluke @GreggVan Not sure if the EN 301 549 is making changes to the quoted WCAG and WCAG2ICT language for the December publication of WCAG, but this issue and it's PR can help you find exactly what changed in the December 2024 WCAG 2.2 and make it easier to incorporate into the EN if you plan to do so.  I'll label it with the EN 301 549 tag just in case but be sure to look for it in the closed WCAG2ICT issues list.
